generators/client/prompts.js

```diff
@@ -113,59 +112,28 @@ function askForClientTheme() {
         { value: 'yeti', name: 'Yeti' },
     ];
 
-    const PROMPT = {
+    const bootSwatchChoices = await retrieveBootswatchThemes(self).catch(() =>
```

Comment:
Is it bootSwatch or bootswatch? I'm confused...

Reply:
bootswatch indeed ;), let me fix that ;)